src/components/app/flyouts/settings/settings.js

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
```

Comment:
Why not `import { React,  Component } from 'react';`?

Reply:
that does not work